Tests/Common/Securities/Options/OptionChainProviderTests.cs

```diff
@@ -77,6 +106,32 @@ public void LiveOptionChainProviderReturnsNoDataForInvalidSymbol()
 
             Assert.IsFalse(result.Any());
         }
+
+        [Test]
+        public void LiveOptionChainProviderReturnsFutureOptionData()
+        {
+            var now = DateTime.Now;
+            var december = now.AddMonths(-now.Month).AddMonths(12);
+            var underlyingFuture = Symbol.CreateFuture("ES", Market.CME, december);
+
+            var provider = new LiveOptionChainProvider();
+            var result = provider.GetOptionContractList(underlyingFuture, december);
+
+            Assert.AreNotEqual(0, result.Count());
```

Comment:
I saw this one fail locally 🤔

Reply:
hmmm, it might only fail on Mondays 🤔 - good catch, I'll test some more and maybe make it more resiliant

Reply:
It's working! 👍  suggest we add some more finer test asserts though,

Reply:
hm these tests are set to run in parallel with each other `Parallelizable(ParallelScope.All)` maybe they should be changed to `Parallelizable(ParallelScope.Fixtures)`